test/opt/opextinst_forward_ref_fixup_pass_test.cpp

```diff
@@ -0,0 +1,365 @@
+// Copyright (c) 2024 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "spirv-tools/optimizer.hpp"
+#include "test/opt/pass_fixture.h"
+#include "test/opt/pass_utils.h"
+
+namespace spvtools {
+namespace opt {
+namespace {
+
+using OpExtInstForwardRefFixupPassTest = PassTest<::testing::Test>;
+
+TEST_F(OpExtInstForwardRefFixupPassTest, HealthCheck) {
```

Comment:
A comment about what this test is doing would be useful.

Reply:
Renamed it to something meaningful